src/whylogs/viewer/templates/index-hbs-cdn-all-in-jupyter-bar-chart.html

```diff
@@ -279,7 +279,7 @@ <h1 class="no-responsive__title">Hold on! :)</h1>
             $(document).ready(() => 
               $(".desktop-content").html(`
                 <p style="height: ${$(window).height()}px" class="error-message"> 
-                  Something went wrong. Please try again.
+                    Error: column must be a stringSummary
```

Comment:
I think we shouldn't assume that the user knows what stringSummary is. Also, it's somewhat confusing, because even though the code relies on stringSummary, the clearest distinction between both plots should be between categorical/numerical values.

Maybe something like "Error: feature must be either categorical or textual" ?